release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3390,6 +3390,42 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.14.40
+[id="ocp-4-14-40_{context}"]
+=== RHSA-2024:8697 - {product-title} 4.14.40 bug fix and security update
+
+Issued: 07 November 2024
+
+{product-title} release 4.14.40, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:8697[RHSA-2024:8697] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:8700[RHSA-2024:8700] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.40 --pullspecs
+----
+
+[id="ocp-4-14-40-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the Cloud Credential Operator (CCO) for a cluster installed on {gcp-first} entered a `Degraded=True` if the cluster was not using manual mode credentials and did not have the `backupdr` API enabled. With this release, an update ensure the cluster does not enter a degraded state when the cluster is configured with this environment. (link:https://issues.redhat.com/browse/OCPBUGS-43821[*OCPBUGS-43821*])
+
+* Previously, when you attempted to use the `oc import-image` command to import an image in a {hcp} cluster, the command failed because of access issues with a private image registry. With this release, an update to `openshift-apiserver` pods in a {hcp} cluster resolves names that use the data plane so that the `oc import-image` command now works as expected with private image registries. (link:https://issues.redhat.com/browse/OCPBUGS-43468[*OCPBUGS-43468*])
+
+* Previously for hosted control planes (HCP), a cluster that used mirroring release images might result in existing node pools to use the hosted cluster's operating system version instead of the `NodePool` version. With this release, a fix ensures that node pools use their own versions. (link:https://issues.redhat.com/browse/OCPBUGS-43368[*OCPBUGS-43368*])
```

Comment:
```suggestion
* Previously for {hcp} a cluster that used mirroring release images might result in existing node pools to use the hosted cluster's operating system version instead of the `NodePool` version. With this release, a fix ensures that node pools use their own versions. (link:https://issues.redhat.com/browse/OCPBUGS-43368[*OCPBUGS-43368*])
```